FlowMagic-SDKTests/ScreenFlowTests/ScreenFlowTests.swift

```diff
@@ -0,0 +1,179 @@
+////
+////  ScreenFlowTests.swift
+////  FlowMagic-SDKTests
+////
+////  Created by Kedar Dhere on 3/24/23.
+////
+//
+import XCTest
+import SwiftUI
+@testable import FlowMagic_SDK
+
+func propertiesAreEqual(_ lhs: Any, _ rhs: Any) -> Bool {
+    let lhsMirror = Mirror(reflecting: lhs)
+    let rhsMirror = Mirror(reflecting: rhs)
+
+    for (lhsChild, rhsChild) in zip(lhsMirror.children, rhsMirror.children) {
+        guard "\(lhsChild.value)" == "\(rhsChild.value)" else { return false }
+    }
+    return true
+}
+
+class ScreenFlowTests: XCTestCase {
+     // Test Register Screen and Get Screen
+    func testRegisterScreens() {
```

Comment:
Your code comment is in front of a function and so should, presumably, be a docstring.  Docstrings start with `///`, not `//`.  (Apple's tooling, including the IDE, has some support for docstrings, so it matters in practical terms.)  Also, a function docstring would describe what the function does, so would need to rephrased (and should be a sentence and one that ends in a period as Apple's own docs do).

Reply:
I've updated the comment to be a proper docstring using /// and rephrased it to provide a clearer description of the function's purpose, following Apple's documentation conventions.